src/main/java/frc/team2412/robot/commands/shooter/ShooterHoodSetAngleCommand.java

```diff
@@ -0,0 +1,33 @@
+package frc.team2412.robot.commands.shooter;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystem.ShooterSubsystem;
+import frc.team2412.robot.subsystem.ShooterVisionSubsystem;
+
+public class ShooterHoodSetAngleCommand extends CommandBase {
+    private final ShooterSubsystem shooter;
+    private final DoubleSupplier angleSupplier;
+
+    public ShooterHoodSetAngleCommand(ShooterSubsystem shooter, double angle) {
+        this.shooter = shooter;
+        this.angleSupplier = () -> angle;
+    }
+
+    public ShooterHoodSetAngleCommand(ShooterSubsystem shooter, ShooterVisionSubsystem vision) {
+        this.shooter = shooter;
+        this.angleSupplier = null; // TODO: Interface with ShooterVisionSubsystem
```

Comment:
For this class and `ShooterTurretUpdateAngleCommand`, I would recommend putting the supplier as a constructor parameter. This would remove the need for two constructors and remove the dependency on `ShooterVisionSubsystem`.

I'm also not sure why `ShooterVisionSubsystem` has to be a subsystem, as several examples I've seen have sensors in a separate directory. However, I don't think the team has agreed one way or another on where they should go, and the WPILib documentation for [subsystems](https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Subsystem.html) says they can be sensors, so you'll probably be fine either way.